skaffold.yaml

```diff
@@ -145,6 +145,14 @@ deploy:
             - name: "node-1"
               secret: "p@$swr0d44"
           extraEnv:
+            - name: STATSD_ENABLED
+              value: "false"
+            - name: STATSD_HOST
+              value: "graphite-0.graphite.monitoring.svc.cluster.local"
```

Comment:
Host value with `svc.cluster.local` is strange here, how do you deploy statsd ?

Reply:
using another private repo